From: Petter Reinholdtsen <pere@hungry.com>
Date: Sun, 8 May 2016 22:22:41 +0200
Subject: Add MIME types imported from Totem

Some of them are commented, until we know more about them.

Signed-off-by: Jean-Baptiste Kempf <jb@videolan.org>
---
 share/vlc.desktop.mimetypes | 63 ++++++++++++++++++++++++++++++++++++++++++++-
 1 file changed, 62 insertions(+), 1 deletion(-)

diff --git a/share/vlc.desktop.mimetypes b/share/vlc.desktop.mimetypes
index c762772..e2881ba 100644
--- a/share/vlc.desktop.mimetypes
+++ b/share/vlc.desktop.mimetypes
@@ -5,9 +5,11 @@
 application/ogg                  # IANA-registered
 application/x-ogg
 audio/ogg                        # IANA-registered
+audio/x-vorbis
 audio/x-vorbis+ogg
 video/ogg                        # IANA-registered
 video/x-ogm+ogg
+video/x-theora+ogg
 
 # Microsoft audio and video
 audio/x-ms-asf
@@ -17,10 +19,14 @@ audio/x-ms-wma
 audio/x-pn-windows-acm
 video/divx
 video/msvideo
+video/vnd.divx
 video/x-avi
 video/x-ms-asf
 video/x-ms-asf-plugin
+video/x-ms-asx
+video/x-ms-wm
 video/x-ms-wmv
+video/x-ms-wmx
 video/x-ms-wvx
 video/x-msvideo
 
@@ -34,10 +40,12 @@ audio/x-realaudio
 video/vnd.rn-realvideo
 
 # MPEG related audio, video and transport
-application/mpeg4-iod 	    	# IANA-registered
+application/mpeg4-iod            # IANA-registered
 application/mpeg4-muxcodetable
+application/x-extension-m4a
 application/x-extension-mp4
 audio/m4a
+audio/mp2
 audio/mp4                        # IANA-registered
 audio/mpeg                       # IANA-registered
 audio/x-m4a
@@ -51,6 +59,7 @@ video/mpeg-system
 video/x-m4v
 video/x-mpeg
 video/x-mpeg-system
+video/x-mpeg2
 
 # FLAC lossless audio
 application/x-flac
@@ -69,6 +78,8 @@ audio/webm
 # RTSP related
 audio/3gpp
 audio/3gpp2
+audio/AMR                        # IANA-registered
+audio/AMR-WB                     # IANA-registered
 audio/amr
 audio/amr-wb
 video/3gpp
@@ -88,6 +99,7 @@ x-content/audio-player
 
 # Playlists / text/xml list with URLs
 application/xspf+xml
+audio/mpegurl
 audio/x-mpegurl
 audio/x-scpls
 video/vnd.mpegurl                # IANA-registered
@@ -96,9 +108,12 @@ video/vnd.mpegurl                # IANA-registered
 video/dv
 
 # Apple QuickTime
+application/x-quicktime-media-link
+application/x-quicktimeplayer
 video/quicktime                  # IANA-registered
 
 # IFF related formats
+audio/x-aiff
 audio/x-pn-aiff
 video/x-anim
 
@@ -106,6 +121,7 @@ video/x-anim
 video/x-nsv
 
 # Autodesk animation format
+video/fli
 video/flv
 video/x-flc
 video/x-fli
@@ -119,9 +135,54 @@ audio/x-wav
 
 # Adobe Flash player related
 application/x-shockwave-flash
+application/x-flash-video
 
 # SHOUTcast 2
 misc/ultravox
 
 # RealPix
 image/vnd.rn-realpix
+
+# From the totem desktop file in Debian.  These need more work.
+application/mxf
+#application/ram
+application/sdp
+#application/smil
+#application/smil+xml
+application/vnd.apple.mpegurl
+application/vnd.ms-asf
+application/vnd.ms-wpl
+#application/x-netshow-channel
+#application/x-shorten
+#application/x-smil
+audio/ac3
+#audio/basic
+#audio/midi
+#audio/prs.sid
+audio/x-ape
+audio/x-gsm
+audio/x-it
+audio/x-mod
+audio/x-musepack
+audio/x-s3m
+#audio/x-sbc
+audio/x-speex
+#audio/x-stm
+audio/x-tta
+audio/x-wavpack
+audio/x-xm
+#image/x-pict
+#text/google-video-pointer
+#text/x-google-video-pointer
+video/3gp
+#video/vivo
+#video/vnd.vivo
+#video/x-flic
+#video/x-totem-stream
+x-scheme-handler/icy
+x-scheme-handler/icyx
+x-scheme-handler/mmsh
+#x-scheme-handler/net
+#x-scheme-handler/pnm
+x-scheme-handler/rtp
+#x-scheme-handler/uvox
